### PR TITLE
Small changes to swedish translation

### PR DESCRIPTION
**Pull request by Mr-Bajs**
retry

### CODE REVIEW

**Comment by AJenbo** [comment]
@SimonCeder or @john-tornblom would you be willing to review this update?

File: Translations/sv.po [hunk 1]
```diff
@@ -2063,7 +2063,7 @@ msgstr "Köttyxa"
 
 #: Source/itemdat.cpp:60 Source/itemdat.cpp:434
 msgid "The Undead Crown"
-msgstr "De Vandödas Krona"
+msgstr "De Odödas Krona"
```
Comment (john-tornblom) [inline Translations/sv.po:2066]: This is fine (according to PS1 title)

File: Translations/sv.po [hunk 2]
```diff
@@ -2560,7 +2560,7 @@ msgstr "Spikklubba"
 
 #: Source/itemdat.cpp:194
 msgid "Flail"
-msgstr "Slaga"
+msgstr "Stridsgissel"
```
Comment (john-tornblom) [inline Translations/sv.po:2563]: PS1 title translates to Slaga (whatever that is)

File: Translations/sv.po [hunk 3]
```diff
@@ -11463,7 +11463,7 @@ msgstr "Krögaren Ogden"
 
 #: Source/towners.cpp:110
 msgid "Wounded Townsman"
-msgstr "Skadad Stadsbo"
+msgstr "Skadad Bybo"
```
Comment (john-tornblom) [inline Translations/sv.po:11466]: PS1 title translates this to "Sårad Stadsbo"

**Review by john-tornblom — COMMENTED** [review]
The PS1 title translates "town" to "stad", so those changes are not correct (I only commented a a couple of them).

The fix for "undead" seems appropriate, but not sure if we should be true to the PS1 title and include inconsistencies between odöd and ickedöd.

File: Translations/sv.po [hunk 4]
```diff
@@ -1087,7 +1087,7 @@ msgstr ""
 
 #: Source/cursor.cpp:220
 msgid "Town Portal"
-msgstr "Stadsportal"
+msgstr "Byportal"
```
[the remark this answers is not in the document]
Comment (john-tornblom) [reply Translations/sv.po:1090]: I agree that "by" (village in English) is a much more appropriate description than "stad" (which can mean both city and town in English) . My only concern is we might get inaccurate subtitles for the Swedish audio available on the PS1 disc (which I use when I play the game). Having that said, the current subtitles used by devilutionX is not at all accurate with the Swedish audio, so not an issue right now.

@AJenbo, you decide on how true we should be to the original PS1 game. I would use "by" for now, and if someone steps up and makes accurate subtitles for the Swedish audio, we can switch back.

[the remark this answers is not in the document]
Comment (AJenbo) [reply Translations/sv.po:1090]: Having the text match the spoken words sounds like a good idea, maybe unless there is something distinctly broken about it. I think even Star Wars does not have the subtitles align with the spoken words.

**Pull request merged**